apps/style/netsim/style.scss

```diff
@@ -268,35 +270,31 @@ svg {
       }
     }
 
-    div.panel_body {
+    div.panel-body {
       border-left: solid darkgray thin;
       border-right: solid darkgray thin;
       border-bottom: solid darkgray thin;
-      border-top: solid $code_org_pacific_blue thin;
+      border-top: solid $code-org-pacific-blue thin;
       border-radius: 0 0 $panel_border_radius $panel_border_radius;
     }
 
+    // Minimized panel, contents are hidden.
+    &.minimized {
+      h1 {
+        border-radius: $panel_border_radius;
```

Comment:
Name of variable `panel_border_radius` should be written in all lowercase letters with hyphens instead of underscores